src/plugins/analytics.js

```diff
@@ -0,0 +1,54 @@
+import ExecutionEnvironment from '@docusaurus/ExecutionEnvironment'
+
+export default (function () {
+  if (!ExecutionEnvironment.canUseDOM) {
+    return null
+  }
+
+  // Emit events for all github link clicks
+  function anchorListener(event) {
+    let element = event.target
```

Comment:
Is there any strange case where this can be undefined?

Reply:
I don't believe so